File: src/ChickenMeat.cpp
#include "../include/ChickenMeat.hpp"

//Constructor menerima harga, dan mempassing producer untuk superclass.
ChickenMeat::ChickenMeat(int _price, string _producer){
	initializeFarmProduct(_producer);
	initializeProduct(_price, FARMPRODUCT, CHICKEN_MEAT, '?');
}

File: Java/src/main/java/engisubs/gameobject/product/farm_product/meat/ChickenMeat.java
package main.java.engisubs.gameobject.product.farm_product.meat;

public class ChickenMeat extends Meat{
    public ChickenMeat(int _price, String _producer){
        super.initializeFarmProduct(_producer);
        super.initializeProduct(_price, ProductType.FARMPRODUCT, GameObjectType.CHICKEN_MEAT, "Daging Segar Ayamosaurus Rex");
    }
}